gpkit/constraints/tight.py

```diff
@@ -17,9 +17,9 @@ def process_result(self, result):
         for constraint in self.flat():
             if isinstance(constraint, (PosynomialInequality,
                                        SignomialInequality)):
-                leftsubbed = constraint.left.sub(variables)
-                rightsubbed = constraint.right.sub(variables)
-                rel_diff = abs(1-(leftsubbed/rightsubbed).value)
+                leftsubbed = constraint.left.sub(variables).value
+                rightsubbed = constraint.right.sub(variables).value
+                rel_diff = abs(1 - leftsubbed/rightsubbed)
```

Comment:
or do we need separate `process_result` methods for processing straight out of the solver vs processing the packaged result post-check-solution?

I can imagine use cases for both. But usually I think the solution should be checked before any processing is done.